src/main/java/io/lettuce/core/ZAddArgs.java

```diff
@@ -109,6 +132,28 @@ public ZAddArgs ch() {
         return this;
     }
 
+    /**
+     * Only update existing elements if the new score is greater than the current score. This flag doesn't prevent adding new elements.
+     *
+     * @return {@code this} {@link ZAddArgs}.
+     */
+    public ZAddArgs gt() {
+
+        this.gt = true;
```

Comment:
ideally, we set `lt` to `false` to not have a state where `lt` and `gt` are both set to true if someone calls `lt().gt()`.